### DISCUSSION TITLE
Segmentation fault issue

### DISCUSSION THREAD

**Discussion by Reaperbruhsans** [community/community#28220] [Codespaces]
Recently, I've tried to run the code I've written on my codespace, however, when I have done that, I've come across with an error that says;

**Segmentation fault (Core dumped)**

I would like to know what the issue is, thank you for the assistance.

**Comment by atuschek0** [comment]
Here's a good resource for this https://www.geeksforgeeks.org/core-dump-segmentation-fault-c-cpp/ . The article states "Core Dump/Segmentation fault is a specific kind of error caused by accessing memory that “does not belong to you.”"

**Reply by Reaperbruhsans** [reply]
Yes, I've seen that description. 🤔
